AsyncDisplayKit/Private/ASLayoutTransition.h

```diff
@@ -47,7 +47,7 @@
 /**
  * Insert and remove subnodes that where added or removed between the previousLayout and the pendingLayout
```

Comment:
Yes, this is basically commitAnimations. @maicki, `commitTransition` seems like a winner.